flink-end-to-end-tests/test-scripts/test_streaming_kinesis.sh

```diff
@@ -27,12 +27,16 @@ export AWS_ACCESS_KEY_ID=flinkKinesisTestFakeAccessKeyId
 export AWS_SECRET_KEY=flinkKinesisTestFakeAccessKey
 
 KINESALITE_PORT=4567
+KINESALITE_HOST=kinesalite-container
+KINESALITE_NETWORK=some
```

Comment:
what's this about? probably deserves a separate commit

Reply:
I'll revert this change. These changes are leftovers of my attempt to make this test pass in a docker-in-docker scenario.